doc_source/aws-properties-ec2-security-group-ingress.md

```diff
@@ -351,3 +351,57 @@ SGPing:
       ToPort: -1
       CidrIp: 10.0.0.0/24
 ```
+
+### Allow Access from a Security Group within a Peered VPC
+
+In some cases, it may become necessary to allow specific traffic from a targeted originating (source) security group in a different AWS Account to specific ports on the target. An example of this scenario could include a one-way security scanning resource that exists in a separate AWS Account that needs to run diagnostics into a peered account. The following snippet creates a security group that allows this access on Port 80 from the client side, ensuring only desired resources are scanned.
+
+#### JSON<a name="aws-resource-ec2-securitygroupingress-example-4.json"></a>
+
+```
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Resources": {
+    "SecurityScanGroup": {
+      "Type": "AWS::EC2::SecurityGroup",
+      "Properties": {
+        "VpcId": "vpc-e063f789",
```

Comment:
This looks like a legitimate VPC ID, suggest obfuscating as you did for the `SecurityScanGroupIngress` resource.

Reply:
@rjlohan I am happy to do that - this VPC ID is referenced earlier in this guide, which is why I felt comfortable including it in the PR. Thoughts?

Reply:
Fair enough, leave as is then!